miso-web/src/main/webapp/scripts/library_ajax.js

```diff
@@ -130,7 +130,14 @@ var Library = Library || {
 };
 
 Library.ui = {
-
+  updateConcentrationUnits: function(concentrationUnits) {
```

Comment:
The'res a lot of duplication here, so it might make sense to abstract them into something like `Utils.ui.updateConcentrationUnits` in `lims.js`